omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ObsController1_9Test.java

```diff
@@ -84,6 +87,38 @@ public void shouldGetAll() throws Exception {
 	}
 	
 	@Test
+	public void shouldGetallIncludingVoidedObs() throws Exception {
+		
+		executeDataSet("encounterWithObsGroup1_9.xml");
+		
+		MockHttpServletRequest getAllNonVoidedObsRequest = newGetRequest(getURI());
```

Comment:
Is getAllNonVoidedObsRequest a function or variable?
The same applies to the others.